code/game/controllers/game.py

```diff
@@ -36,16 +38,24 @@ def handle_events(self) -> None:
                 self._enemies.remove_enemy(enemy.id)
 
             elif enemy.distance < PLANET_DISTANCE:
-                self._planet.get_damage(enemy.damage)
-                self._enemies.remove_enemy(enemy.id)
+                try:
+                    self._planet.get_damage(enemy.damage)
+                    self._enemies.remove_enemy(enemy.id)
+                except DamageLimitException:
+                    self._status = GameStatus.END
 
     @property
     def state(self) -> dict:
-        return {
-            'cat': self._cat.state,
-            'enemies': self._enemies.state,
-            'planet': self._planet.state,
-        }
+        state = {'game': {'status': self._status}}
```

Comment:
такие вложенные структуры давай столбиком делать, чтобы было легче читать?
```suggestion
        state = {
            'game': {
                'status': self._status,
            },
        }
```
или лучше так:
```suggestion
        state = {
            'game': {'status': self._status},
        }
```